installer/bundle/bundle_skel_AIX.sh

```diff
@@ -134,13 +192,52 @@ pkg_rm() {
 pkg_upd() {
     pkg_filename=$1
     pkg_name=$2
+    pkg_allowed=$3
 
     echo "----- Updating package: $pkg_name ($pkg_filename) -----"
 
+    if [ -z "${forceFlag}" -a -n "$pkg_allowed" ]; then
```

Comment:
Yeah, but I think this is correct. While it would be improper to call pkg_upd without $pkg_allowed, if it was called (say because we were forcing or whatever), we don't want the test at line 200 if $pkg_allowed is blank. So I would argue that, if pkg_allowed is not passed, this does exactly what you want. And if pkg_allowed is passed, you do exactly what you want. Right?